backend/src/Designer/Services/Implementation/GiteaAPIWrapper.cs

```diff
@@ -201,6 +201,46 @@ public async Task<IList<RepositoryClient.Model.Repository>> GetOrgRepos(string o
             return repos;
         }
 
+        public async Task<ListviewServiceResource> MapServiceResourceToListViewResource(string repo, ServiceResource serviceResource)
+        {
+            string developer = AuthenticationHelper.GetDeveloperUserName(_httpContextAccessor.HttpContext);
+            ListviewServiceResource listviewResource = new ListviewServiceResource { Identifier = serviceResource.Identifier, Title = serviceResource.Title };
+            string resourceFolder = serviceResource.Identifier;
+
+            HttpResponseMessage response = await _httpClient.GetAsync($"repos/{developer}/{repo}/contents/{resourceFolder}/{serviceResource.Identifier}_resource.json");
+
+            if (response.StatusCode == HttpStatusCode.OK)
+            {
+                ContentsResponse contentsResponse = await response.Content.ReadAsAsync<ContentsResponse>();
+                response = await _httpClient.GetAsync($"repos/{developer}/{repo}/git/commits/{contentsResponse.LastCommitSha}");
+                if (response.StatusCode == HttpStatusCode.OK)
+                {
+                    GiteaCommit lastCommit = await response.Content.ReadAsAsync<GiteaCommit>();
+                    listviewResource.LastChanged = DateTime.Parse(lastCommit.Created);
+                }
+
+                HttpResponseMessage responseFromCommits = await _httpClient.GetAsync($"repos/{developer}/{repo}/commits");
```

Comment:
Little confused here. What does developer mean in context here? Are we calling Gitea API here? Cant it be org?